src/codegen/util/cc_hash_table.cpp

```diff
@@ -79,7 +79,7 @@ char* CCHashTable::StoreTuple(uint64_t hash, uint32_t size) {
 // Clean up any resources this hash table has
 //===----------------------------------------------------------------------===//
 void CCHashTable::Destroy() {
-  LOG_DEBUG("Cleaning up hash table with %ld entries ...", num_elements_);
+  LOG_DEBUG("Cleaning up hash table with %llu entries ...", (unsigned long long) num_elements_);
```

Comment:
Failed in Mac 
```
/Users/angli/Documents/peloton_mac/peloton/src/codegen/util/cc_hash_table.cpp:82:60: error: 
      format specifies type 'unsigned long' but the argument has type 'uint64_t'
      (aka 'unsigned long long') [-Werror,-Wformat]
  LOG_DEBUG("Cleaning up hash table with %lu entries ...", num_elements_);
                                         ~~~               ^~~~~~~~~~~~~
                                         %llu
/Users/angli/Documents/peloton_mac/peloton/src/include/common/logger.h:124:32: note: 
      expanded from macro 'LOG_DEBUG'
  ::fprintf(LOG_OUTPUT_STREAM, __VA_ARGS__);                                  \
                               ^~~~~~~~~~~

```
The reason is that uint64_t is unsigned long long in Mac but unsigned long in Linux